test/unit/serialization/ReferenceObjects.js

```diff
@@ -88,12 +88,28 @@ to.aLocalTime = new LocalTime(11, 22, 41, 123456789);
 to.aLocalDateTime = new LocalDateTime(to.aLocalDate, to.aLocalTime);
 to.aOffsetDateTime = new OffsetDateTime(to.aLocalDateTime, 64800);
 
+to.Objects = [to.AnInnerPortable, to.aNULL, to.aBigDecimal, to.aShort];
```

Comment:
All of the `to.XXX` stuff starts with a lower case `a`m but `to.AnInnerPortable` starts with an uppercase one. It may be renamed to `to.anInnerPortable` for consistency.

Reply:
I think names are retrieved from java classes and the java class is named AnInnerPortable. Therefore, if we will put an article it should be anAnInnerPortable, anInnerPortable won't work because after stripping the article it will be InnerPortable which does not exist. I will use anAnInnerPortable